api-tester/src/main/java/com/revenuecat/apitester/java/PurchasesAPI.java

```diff
@@ -76,11 +76,29 @@ static void check(final Purchases purchases,
         purchases.getOfferings(receiveOfferingsListener);
         purchases.getProducts(productIds, productResponseListener);
         purchases.purchaseProduct(activity, storeProduct, upgradeInfo, purchaseChangeListener);
+        purchases.purchaseProduct(activity, storeProduct, upgradeInfo, true,  purchaseChangeListener);
         purchases.purchaseProduct(activity, storeProduct, makePurchaseListener);
+        purchases.purchaseProduct(activity, storeProduct, true, makePurchaseListener);
         purchases.purchasePackage(activity, packageToPurchase, upgradeInfo, purchaseChangeListener);
+        purchases.purchasePackage(
+                activity,
+                packageToPurchase,
+                upgradeInfo,
+                true,
+                purchaseChangeListener
+        );
```

Comment:
talked offline, but I agree we have to do a builder, this is getting very complex. I think it's a good moment to deprecate all methods and clean this up.